### PR TITLE
Enhancement: Dont crash the UI on RJSF Form Errors

### PR DESCRIPTION
**Pull request by Abhishek-kumar09**
**Notes for Reviewers**
The RJSF wont crash further on any invalid schema.

Screenshot 2023-07-27 at 2 55 12 AM

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/ErrorBoundary.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { Button } from "@material-ui/core";
+import React from "react";
+
+/**
+ * ErrorBoundary is a React component that catches JavaScript errors in its child components and renders a fallback UI when an error occurs.
```
Comment (leecalcote) [inline ui/components/ErrorBoundary.js:5]: Same one used in `meshery-extensions`?

File: ui/components/ErrorBoundary.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  /** You can render any custom fallback UI */
+  render() {
+    if (this.state.hasError) {
+      return (
+        <div className="alert alert-danger">
+          <p>Couldn&apos;t open RJSF Form, the following error was encountered:</p>
+          <pre>{this.state.error.message}</pre>
+          <Button color="primary" variant="contained" onClick={this.resetErrorBoundary}>
+            Refresh Form
+          </Button>
```
Comment (leecalcote) [inline ui/components/ErrorBoundary.js:32]:
```suggestion
          <p>Couldn&apos;t open form. Encountered the following error:</p>
          <pre>{this.state.error.message}</pre>
          <Button color="primary" variant="contained" onClick={this.resetErrorBoundary}>
            Refresh Form
          </Button>
```

**Review by leecalcote — APPROVED** [review]
LGTM

File: ui/components/ErrorBoundary.js (hunk 1 above)
Comment (Abhishek-kumar09) [reply ui/components/ErrorBoundary.js:5]: This component usage will take affect on the Meshery-extensions as well.
With your comment, I just saw, that meshery-extensions is also using an errorBoundary class. Although its not same. It is using this package: https://www.npmjs.com/package/react-error-boundary

The behaviour is same though. I can just use that as well

**Pull request merged**